ephypype/nodes/utils.py

```diff
@@ -73,3 +73,35 @@ def create_datagrabber(data_path, template_path, template_args):
     datasource.inputs.sort_filelist = True
 
     return datasource
+
+
+def get_frequency_band(freq_band_names, freq_bands):
+    """"Create node to get frequency band of interest.
+
+    Parameters
+    ----------
+    freq_band_names : list of str
+        The frequency band names
+    freq_bands : list of list
+        The interval of frequency bands
+
+    Returns
+    -------
+    frequency_node : instance of pe.Node
+        The node which recover the frequency band of interest.
+    """
+
+    from ephypype.aux_tools import get_freq_band
```

Comment:
use relative imports. Never import from `ephypype` from within ephypype. Use something like:

```py
from .aux_tools import get_freq_band
```

and put this at the top of the file

Reply:
> use relative imports. Never import from `ephypype` from within ephypype. Use something like:

Yes, you told me and in #56 I'm reorganizing all! :)

Reply:
> and put this at the top of the file

From the [link](https://nipype.readthedocs.io/en/latest/users/function_interface.html) @davidmeunier79 shared with us it seems if a function it's used in a Node the import has to be inside